homeassistant/components/squeezebox/browse_media.py

```diff
@@ -78,83 +94,209 @@
     "Playlists": MediaType.PLAYLIST,
     "Genres": MediaType.GENRE,
     "Favorites": None,  # can only be determined after inspecting the item
+    "Apps": MediaClass.APP,
+    "Radios": MediaClass.APP,
+    "App": None,  # can only be determined after inspecting the item
     "New Music": MediaType.ALBUM,
+    MediaType.APPS: MediaType.APP,
+    MediaType.APP: MediaType.TRACK,
 }
 
 
+@dataclass
+class Squeezebox_maps:
```

Comment:
Classes should follow Pascal Casing, so this would be `SqueezeboxMaps`, but I think a better, more descriptive name is:
```suggestion
class BrowseData:
```